relatio/utils.py

```diff
@@ -56,7 +56,13 @@ def split_into_sentences(
         time.sleep(1)
         docs = tqdm(docs)
 
+    # Ready to be submitted now and test for merge
```

Comment:
Not useful for the end-user :)

Reply:
Yeah those were for me, I guess the non useful comments can be removed.